doc-Managing_Infrastructure_and_Inventory/topics/Adding_an_Interface_to_Network_Router.adoc

```diff
@@ -0,0 +1,12 @@
+[[adding-interface-network-routers]]
+=== Adding an Interface to a Network Router  
+
+Interfaces allow you to interconnect routers with subnets. As a result, the router can direct any traffic that instances send to destinations outside of their intermediate subnet.
+
+To add an interface to a network router:
+
+. Navigate to menu:Networks[Network Routers].
+. Select a network router from the list view. 
+. Click  image:1847.png[Configuration] (*Configuration*), then  image:1851.png[Add Interface to selected Router] (*Add Interface to selected Router*).
```

Comment:
I had a quick look at the 6 Feb build, UI shows "Add Interface to this Router". Also, as per convention, we avoid using drop-down menu and either just say "from the list" or "from the *Subnet* list* - worth bringing this up in our meeting for a quick discussion if required.